src/utils/check-pr/check-pr.py

```diff
@@ -277,7 +277,15 @@ def run(self, *, merge_on_success: bool = False) -> None:
 
 
 def main() -> None:
-    default_instance = "pr-check-%s" % uuid.uuid4().hex
+    # Pull out a username to include in the resource group name
+    # USER is used in wsl2, USERNAME is used under windows
+    if "USER" in os.environ:
+        user=os.environ["USER"]
+    elif "USERNAME" in os.environ:
+        user=os.environ["USERNAME"]
+    else:
+        user=""
```

Comment:
In the failure case, this will resolve to pr-check--GUID, which is probably not undesired.

Given this requires the `az` command be logged in, perhaps `az ad signed-in-user show --query userPrincipalName` and remove everything after `@` would be better.

Reply:
If you're OK requiring service accounts that deploy using check-pr, then this would work without having to split:  `az ad signed-in-user show --query mailNickname`